### DISCUSSION TITLE
[DropdownMenu.SubContent] Is there a way to make it 100% headless css?

### DISCUSSION THREAD

**Discussion by azabraao** [radix-ui/primitives#1923] [Help]
I see this component comes with some inline css

<img width="319" alt="image" src="https://user-images.githubusercontent.com/17601387/216075625-a39c01e3-436a-4eff-a2ad-c46b8aa4e339.png">

Is there a way to customize or opt for not using those rules?

**Comment by joaom00** [comment]
Hi @azabraao,

Without this it is not possible to position the content correctly, why do you want to customize them?

**Reply by azabraao** [reply]
@joaom00 I want to center the submenu for mobile, instead of having it to the side. Is that possible?

**Reply by joaom00** [reply]
I'm afraid not. Have [an open discussion](https://github.com/radix-ui/primitives/issues/1749) about the same concern.

**Reply by azabraao** [reply]
thanks for sharing the link
